pnet_sys/Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["networking", "datalink", "ethernet", "raw"]
 categories = ["network-programming"]
 
 [dependencies]
-libc = "0.2"
+libc = ">=0.2.34"
```

Comment:
You could just set the version to `"0.2.34"`. If you read the docs on version specification you see that this is equal to `"^0.2.34"` [caret requirements](http://doc.crates.io/specifying-dependencies.html#caret-requirements). That would allow all 0.2.xyz versions of libc that has patch version *at least 34*.

When using inequality requirements like this I think everything that is above `0.2.34` is accepted. This means any future `0.3.0` release of libc would be pulled in automatically, while in reality it has a low chance of working. So specifying the dependency like you have done here might be risky for that reason.

Reply:
I know this project has used `>=` dependencies before. But that was mostly for internal crates and used with a zero: `>=0`. We are working towards getting rid of those, which is why I don't want to introduce more of them.